src/Node.cs

```diff
@@ -298,7 +298,8 @@ public static rcl_node create_native_node (string name)
 		{
 			rcl_node_t node = rcl_get_zero_initialized_node ();
 			rcl_node_options_t default_options = rcl_node_get_default_options ();
-			int ret = rcl_node_init (ref node, name,ref default_options);
+            string ns = "";
```

Comment:
Please add call the string namespace and add it as a parameter with "" as default to the method.